starlight/static/js/App.js

```diff
@@ -0,0 +1,3 @@
+if (module && module.hot) {
```

Comment:
At the top of this file, we will need to link in our JS dependencies, like jQuery and Semantic UI, so Webpack can go find them and build them into our assets. For example, for jQuery we can do:

`import $ from 'jquery';`